Sources/Rules/indent.swift

```diff
@@ -0,0 +1,793 @@
+//
+//  indent.swift
+//  SwiftFormat
+//
+//  Created by Cal Stephens on 7/27/24.
```

Comment:
If we decide to land this change and move all of the rules, I wouldn't mind preserving the original authorship info by checking the git history. 

For example, for `indent`:

```suggestion
//  Created by Nick Lockwood on 8/22/16.
```